src/OrchardCore.Themes/TheTheme/Recipes/saas.recipe.json

```diff
@@ -47,6 +48,35 @@
         "Controller": "Home",
         "Area": "TheTheme"
       }
+    },
+    {
+      "name": "FeatureProfiles",
+      "FeatureProfiles": {
+        "limited-tenant": {
+          "FeatureRules": [
```

Comment:
```json
Move bracket to a new line

"FeatureRules":
  [
    {
      "Rule": "Exclude",
      "Expression": "OrchardCore.Contents.FileContentDefinition"
    },
```

Reply:
The bracket style in the code is used in all recipes too. This is because, unlike in C#, we use same-line brackets for everything JS.